.github/workflows/lint.yaml

```diff
@@ -67,6 +67,11 @@ jobs:
       - name: Run Helm linter
         run: |
           helm lint ${{ env.HELM_CHARTS_DIR }}
+      - name: Run TypeSpec format check
```

Comment:
Do we need to update any documentation for this? We're adding a new requirement for contributors.

Reply:
I will check this and update necessary docs.